vmware-base/src/main/java/com/cloud/hypervisor/vmware/mo/VirtualMachineMO.java

```diff
@@ -3292,6 +3292,21 @@ public int getVirtualHardwareVersion() throws Exception {
         return vhOption.getHwVersion();
     }
 
+    /**
+     * Return a hardware version string in the format expected by Vmware
+     * Format: "vmx-DD" where DD represents the hardware version number
+     */
+    public String getVmxFormattedVirtualHardwareVersion() throws Exception {
```

Comment:
Can you add a quick unit test for the same @nvazquez ?

Reply:
Done thanks